apis/python/tests/test_platform_config.py

```diff
@@ -76,34 +75,78 @@ def test_platform_config(adata):
 
 def test__from_platform_config__admits_ignored_config_structure():
     try:
-        TileDBCreateOptions.from_platform_config(
+        tco.TileDBCreateOptions.from_platform_config(
             dict(
-                tiledb=dict(create=TileDBCreateOptions(), future_option=dict(foo="1")),
+                tiledb=dict(
+                    create=tco.TileDBCreateOptions(), future_option=dict(foo="1")
+                ),
                 not_tiledb=dict(read=dict(buffer_size="128")),
             )
         )
     except Exception as e:
         pytest.fail(f"unexpected exception {e}")
 
 
+def test__from_platform_config__admits_ignored_options():
+    tco.TileDBCreateOptions.from_platform_config(
+        {"tiledb": {"create": {"zzz_future_option": "hello"}}}
+    )
+
+
 def test__from_platform_config__admits_plain_dict():
-    tdb_create_options = TileDBCreateOptions.from_platform_config(
+    tdb_create_options = tco.TileDBCreateOptions.from_platform_config(
         {"tiledb": {"create": {"dims": {"soma_dim_0": {"tile": 6}}}}}
     )
     assert tdb_create_options.dim_tile("soma_dim_0") == 6
 
 
-def test__from_platform_config__admits_tiledb_create_options_object():
-    tdb_create_options = TileDBCreateOptions.from_platform_config(
+def test__from_platform_config__admits_create_options_in_shallow_dict():
+    tdb_create_options = tco.TileDBCreateOptions.from_platform_config(
+        {"tiledb": tco.TileDBCreateOptions(dims={"soma_dim_0": {"tile": 6}})}
+    )
+    assert tdb_create_options.dim_tile("soma_dim_0") == 6
+
+
+def test__from_platform_config__admits_create_options_in_dict():
```

Comment:
```suggestion
def test__from_platform_config__admits_create_options_in_fully_specified_dict():
```
not sure of my wording choice, but suggest something similar to help reader

Reply:
updated these and other names.